### PR TITLE
imp(cli): separate CLI commands to it's own section and cleanup configurations

### PR DESCRIPTION
**Pull request by Vvaradinov**
Proposed we split the CLI commands into it's own section. Additional cleanup on the configuration section.

### CODE REVIEW

File: docs/protocol/evmos-cli/configuration.mdx [hunk 1]
```diff
@@ -144,28 +138,137 @@ evmosd config
...
+Another benefit of Websocket is that after the handshake procedure, the overhead of individual messages is low, making it good for sending high number of requests.
+The WebSocket Server can be enabled from the `app.toml`
+
+```toml
+# Address defines the EVM WebSocket server address to bind to.
+ws-address = "0.0.0.0:8546"
+```
 
-A list of commonly used flags of `evmosd` is listed below:
+## ETH Tx Indexer
```
Comment (fedekunze) [inline docs/protocol/evmos-cli/configuration.mdx:244]: pretty sure we removed this from Evmos

**Review by danburck — COMMENTED** [review]
Thanks @Vvaradinov I reviewed and left some comments. 

Can you also add references in https://deploy-preview-30--dancing-hummingbird-242a98.netlify.app/protocol/evmos-cli to link the reader to the configuration and single node pages (similar to how its done with the [docker section](https://deploy-preview-30--dancing-hummingbird-242a98.netlify.app/protocol/evmos-cli#docker)?)

File: docs/protocol/evmos-cli/configuration.mdx [hunk 2]
```diff
@@ -3,17 +3,19 @@ sidebar_position: 1
 ---
 
 # Configuration
```
Comment (danburck) [inline docs/protocol/evmos-cli/configuration.mdx:5]: nit, missing empty line
```suggestion
# Configuration

```

File: docs/protocol/evmos-cli/configuration.mdx [hunk 3]
```diff
@@ -3,17 +3,19 @@ sidebar_position: 1
...
+The following page will guide you through the configuration of a node and a client.
+The node is used to run the blockchain network, produce blocks and validate transactions.
+The client is used as a gateway to interact with the blockchain network by sending transactions and querying the state.
+Additionally we walk through [running the JSON-RPC server](#running-the-json-rpc-server)
```
Comment (danburck) [inline docs/protocol/evmos-cli/configuration.mdx:9]: nit, missing a full-stop
```suggestion
Additionally we walk through [running the JSON-RPC server](#running-the-json-rpc-server).
```

File: docs/protocol/evmos-cli/single-node.md [hunk 4]
```diff
@@ -164,21 +167,9 @@ You can then stop the node using `Ctrl+C`.
 
 ---------------------------
```
Comment (danburck) [inline docs/protocol/evmos-cli/single-node.md:168]:
```suggestion
```

**Review by danburck — APPROVED** [review]
Thanks @Vvaradinov LGTM
I left some nit comments, feel free to merge

**Pull request merged**